agimus_demos_common/launch/franka_common.launch.py

```diff
@@ -122,13 +123,59 @@ def launch_setup(
         value_type=str,
     )
 
+    xacro_collision_args = {
```

Comment:
Instead of copying it, you could make a list with robot descriptions that are generated in a loop, where one is done with `with_sc` set to `True` and the other set to `False`. There is no need to copy the code since it shares virtually all the lines except one parameter.